docs/languages/python.md

```diff
@@ -33,14 +33,13 @@ To experience Python, create a file (using the [File Explorer](/docs/getstarted/
 print("Hello World")
 ```
 
-The Python extension then provides shortcuts to run Python code in the currently selected interpreter (**Python: Select Interpreter** in the Command Palette):
+The Python extension then provides shortcuts to run Python code using the currently selected interpreter (**Python: Select Interpreter** in the Command Palette), just click the **Run Python File in Terminal** play button in the top-right side of the editor.
 
-- In the text editor: right-click anywhere in the editor and select **Run Python File in Terminal**. If invoked on a selection, only that selection is run.
-- In Explorer: right-click a Python file and select **Run Python File in Terminal**.
+![Using the run python file in terminal button](images/tutorial/run-python-file-in-terminal-button.png)
```

Comment:
You won't be able to reach over to the docs/python images folder from docs/languages and it's better to not share images widely since this can break if topics are moved around.
Can you make a second copy of the run-python-file-in-terminal-button.png under docs/languages/images/python? Thanks

Reply:
Ah, thanks for the catch!